rosdep/python.yaml

```
@@ -6663,6 +6668,10 @@ python3-qt5-bindings-webkit:
   gentoo: ['dev-python/PyQt5[webkit]']
   openembedded: [python3-pyqt5@meta-qt5]
   ubuntu: [python3-pyqt5.qtwebkit]
+python3-qtpy:
```

On Arch python is python3 so we can add it here too.
```suggestion
python3-qtpy:
  arch: [python-qtpy]
```